src/spdx/parser/tagvalue/parser/tagvalue.py

```diff
@@ -0,0 +1,689 @@
+
+    @grammar_rule("pkg_lic_decl : PKG_LICS_DECL license_or_no_assertion_or_none")
+    def p_pkg_license_declared(self, p):
+        self.check_that_current_element_matches_class_for_value(Package, p.lineno(1))
+        set_value(p, self.current_element, argument_name="license_declared")
+
+    @grammar_rule("pkg_lic_ff : PKG_LICS_FFILE license_or_no_assertion_or_none")
+    def p_pkg_license_info_from_file(self, p):
+        self.check_that_current_element_matches_class_for_value(Package, p.lineno(1))
+        if p[2] == SpdxNone() or p[2] == SpdxNoAssertion():
+            self.current_element["license_info_from_files"] = p[2]
+        else:
+            self.current_element.setdefault("license_info_from_files", []).append(p[2])
+
+    @grammar_rule("pkg_lic_conc : PKG_LICS_CONC license_or_no_assertion_or_none")
+    def p_pkg_license_concluded(self, p):
+        self.check_that_current_element_matches_class_for_value(Package, p.lineno(1))
+        set_value(p, self.current_element, argument_name="license_concluded")
+
+    @grammar_rule("pkg_checksum : PKG_CHECKSUM CHECKSUM")
+    def p_pkg_checksum(self, p):
+        self.check_that_current_element_matches_class_for_value(Package, p.lineno(1))
+        checksum = parse_checksum(self.current_element["logger"], p[2], p.lineno(1))
+        self.current_element.setdefault("checksums", []).append(checksum)
+
+    @grammar_rule("verification_code : PKG_VERF_CODE LINE")
+    def p_pkg_verification_code(self, p):
+        self.check_that_current_element_matches_class_for_value(Package, p.lineno(1))
+        if str(p.slice[0]) in self.current_element:
+            self.current_element["logger"].append(f"Multiple values for {p[1]} found. Line: {p.lineno(1)}")
+            return
+        verif_code_regex = re.compile(r"([0-9a-f]{40})\s*(\(excludes:\s*(.+)\))?", re.UNICODE)
+        verif_code_code_grp = 1
+        verif_code_exc_files_grp = 3
+        match = verif_code_regex.match(p[2])
+        if not match:
+            self.current_element["logger"].append(
+                f"Error while parsing {p[1]}: Value did not match expected format. Line: {p.lineno(1)}")
+            return
+        value = match.group(verif_code_code_grp)
+        excluded_files = None
+        if match.group(verif_code_exc_files_grp):
+            excluded_files = match.group(verif_code_exc_files_grp).split(",")
+        self.current_element[str(p.slice[0])] = PackageVerificationCode(value, excluded_files)
+
+    @grammar_rule("files_analyzed : PKG_FILES_ANALYZED LINE")
+    def p_pkg_files_analyzed(self, p):
+        self.check_that_current_element_matches_class_for_value(Package, p.lineno(1))
+        if str(p.slice[0]) in self.current_element:
+            self.current_element["logger"].append(f"Multiple values for {p[1]} found. Line: {p.lineno(1)}")
+            return
+        self.current_element[str(p.slice[0])] = p[2] in ['true', 'True', True]
```

Comment:
`p[2]` is a String, is it not? Strings can't be equal to `True`

Reply:
true